.travis.yml

```diff
@@ -8,14 +8,13 @@ script:
 
 # - xctool -workspace ActionSheetPicker-3.0.xcworkspace -scheme CoreActionSheetPicker build
 # - xctool -workspace ActionSheetPicker-3.0.xcworkspace -scheme ActionSheetPickerTests test
-
 # xcode_workspace: ActionSheetPicker-3.0.xcworkspace
 # xcode_scheme: ActionSheetPickerTests
 # xcode_sdk: iphonesimulator
 
 notifications:
   email:
     recipients:
-      - sky4winder+actionSheetPicker@gmail.com
+      - jki@jki.lv
```

Comment:
🧐 Travis is not worked correctly yet. If you want to setup it - please make a list of recipients